distro/README.md

```diff
@@ -0,0 +1,8 @@
+# Releasing rules_platform
+
+1. Update version in distro/BUILD.bazel,
```

Comment:
This is a good first step, but we should eventually be able to automate this.

Reply:
That's the next thing I'm looking into. `rules_java` has that automation done but I don't know how to set that up yet.